scripts/variantstore/wdl/extract/run_in_hail_cluster.py

```diff
@@ -24,7 +24,12 @@ def wrap(string):
 
 
 def run_in_cluster(cluster_name, account, worker_machine_type, region, gcs_project,
-                   autoscaling_policy, script_path, secondary_script_path, vds_path, temp_path, avro_path):
+                   autoscaling_policy, script_path, secondary_script_path, use_classic_vqsr, vds_path, temp_path, avro_path):
+
+    use_classic_vqsr_flag = ""
+    if use_classic_vqsr:
+        use_classic_vqsr_flag = "--use-classic-vqsr"
+        print("using vets as filter model creation")
```

Comment:
not sure this `print` is needed but if so:
```suggestion
        print("using VQSR for filter model creation")
```